### PR TITLE
Update Sufia and other dependencies

### PR DESCRIPTION
**Pull request by mjgiarlo**

### CODE REVIEW

**Comment by atz** [comment]
What causes the changes to column order in schema.rb?  Is that superficial or semantically important?

**Comment by mjgiarlo** [comment]
@atz Maybe those aren't necessary; I regenerated my db locally.

**Comment by atz** [comment]
Yeah, I suspect unnecessary so therefore unwanted.

**Comment by mjgiarlo** [comment]
Yep, removing and pushing.

**Comment by mjgiarlo** [comment]
Crap. Rebase gone awry. One sec.

File: app/assets/javascripts/application.js [hunk 1]
```diff
@@ -12,6 +12,8 @@
 //
 //= require jquery
 //= require jquery_ujs
+//= require dataTables/jquery.dataTables
```
Comment (jcoyne) [inline app/assets/javascripts/application.js]: why is this necessary given https://github.com/projecthydra-labs/lerna/pull/335/files ?

Comment (mjgiarlo) [inline app/assets/javascripts/application.js]: It wasn't -- rebase went bad.

**Comment by mjgiarlo** [comment]
@atz @jcoyne ready
